src/overlays/actors/ovl_En_Kanban/z_en_kanban.c

```diff
@@ -28,27 +30,1016 @@ const ActorInit En_Kanban_InitVars = {
+};
+
+#include "z_en_kanban_gfx.c"
+
+static f32 sCutAngles[] = {
+    /* CUT_POST   */ 0.50f * M_PI,
+    /* CUT_VERT_L */ 0.00f * M_PI,
+    /* CUT_HORIZ  */ 0.50f * M_PI,
+    /* CUT_DIAG_L */ 0.66f * M_PI,
+    /* CUT_DIAG_R */ 0.34f * M_PI,
+    /* CUT_VERT_R */ 0.00f * M_PI,
+    /*            */ 0.00f * M_PI,
+    /*            */ 0.00f * M_PI,
+};
+
+#include "overlays/ovl_En_Kanban/ovl_En_Kanban.c"
+
+void EnKanban_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    EnKanban* this = THIS;
+    f32 zShift;
+    f32 zShift2;
+    s32 i;
+    Player* player = GET_PLAYER(globalCtx);
+    u8* shadowTex = GRAPH_ALLOC(globalCtx->state.gfxCtx, ARRAY_COUNT(sShadowTexFlags));
+
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+
+    func_8012C28C(globalCtx->state.gfxCtx);
+    func_8012C2DC(globalCtx->state.gfxCtx);
+
+    gSPDisplayList(POLY_OPA_DISP++, object_kanban_DL_000C30);
+
+    if (this->actionState != ENKANBAN_SIGN) {
+        Matrix_InsertTranslation(this->actor.world.pos.x, this->actor.world.pos.y, this->actor.world.pos.z,
+                                 MTXMODE_NEW);
+        Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
+        Matrix_RotateStateAroundXAxis(this->floorRot.x);
+        Matrix_InsertZRotation_f(this->floorRot.z, MTXMODE_APPLY);
+        Matrix_InsertTranslation(0.0f, this->actor.shape.yOffset, 0.0f, MTXMODE_APPLY);
+        Matrix_RotateY(this->actor.shape.rot.y, MTXMODE_APPLY);
+        Matrix_InsertXRotation_s(this->actor.shape.rot.x, MTXMODE_APPLY);
+
+        zShift = fabsf(Math_SinS(this->spinRot.x) * this->pieceHeight);
+        zShift2 = fabsf(Math_SinS(this->spinRot.z) * this->pieceWidth);
+        zShift = CLAMP_MIN(zShift, zShift2);
+        zShift *= -(f32)this->direction;
+
+        Matrix_InsertTranslation(0.0f, 0.0f, zShift, MTXMODE_APPLY);
+        Matrix_InsertXRotation_s(this->spinRot.x, MTXMODE_APPLY);
+        Matrix_RotateY(this->spinRot.z, MTXMODE_APPLY);
+        Matrix_InsertTranslation(this->offset.x, this->offset.y, this->offset.z - 100.0f, MTXMODE_APPLY);
+
+        gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+
+        for (i = 0; i < ARRAY_COUNT(sPartFlags); i++) {
+            if (sPartFlags[i] & this->partFlags) {
+                gSPDisplayList(POLY_OPA_DISP++, sDisplayLists[i]);
+            }
+        }
+    } else {
+        f32 phi_f0 = 0.0f;
+
+        if ((player->transformation == PLAYER_FORM_HUMAN) || (player->transformation == PLAYER_FORM_DEKU)) {
+            phi_f0 = -15.0f;
+        }
+        this->actor.world.pos.y = this->actor.home.pos.y + phi_f0;
+        Matrix_InsertTranslation(0.0f, 0.0f, -100.0f, MTXMODE_APPLY);
+        gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+
+        if (this->partFlags == 0xFFFF) {
+            gSPDisplayList(POLY_OPA_DISP++, D_0405AED0);
```

Comment:
`D_0405AED0` is part of gameplay keep. Should be able to import its object and use the symbol.